src/es/doramasflix/src/eu/kanade/tachiyomi/animeextension/es/doramasflix/Doramasflix.kt

```diff
@@ -350,123 +386,112 @@ class Doramasflix : ConfigurableAnimeSource, AnimeHttpSource() {
         return POST(apiUrl, popularRequestHeaders, body)
     }
 
+    private fun popularMovieRequest(page: Int): Request {
+        val mediaType = "application/json; charset=utf-8".toMediaType()
+        val body = (
+            "{\"operationName\":\"listMovies\",\"variables\":{\"perPage\":32,\"sort\":\"CREATEDAT_DESC\",\"filter\":{},\"page\":$page},\"query\":\"query " +
```

Comment:
Maybe use triple quotes as string delimiter so you don't have to escape the quotes?

Reply:
Months or maybe years ago, when I originally made this extension, I noticed that those "escapes" (and "line breaks") are part of the body,  if I removed the "escapes", when making the request, the API returned an error .